app.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask, render_template, request, redirect
+
```

Comment:
You can create `config-template.yaml` and add `css_framework` property. Then load it to variable and send it to template where it will decide which framework will be used. If it is not in config, set it to bootstrap (default val).

```suggestion

with open("config-template.yaml", "r") as ymlfile:
    cfg = yaml.safe_load(ymlfile)

```